configure.ac

```diff
@@ -164,6 +164,11 @@ AX_CHECK_COMPILE_FLAG([-std=c++11], [
 AC_ARG_ENABLE([qt], AS_HELP_STRING([--enable-qt], [Enable Qt: needed for pulseaudio and jack GUIs]), [], [enable_qt=check])
 AS_IF([test "$enable_qt" != "no"],
   [
+    case $host_os in
+      linux*)
+        PATH="$PATH:`pkg-config --variable=host_bins Qt5Core`"
```

Comment:
comment this please... i have no idea what this is doing

Reply:
hello , I'm studying this [1], while the work it is and hack ! 
I will try improve it today 
[1]
https://autotools.io/pkgconfig/pkg_check_modules.html